### ISSUE TITLE
MVC UI Index.cshtml page generation improvement.

### ISSUE THREAD

**Issue by gdlcf88** [EasyAbp/AbpHelper.CLI#38]
https://github.com/EasyAbp/AbpHelper.CLI/blob/develop/src/AbpHelper/Templates/Crud/Groups/UiRazor/src/%7B%7B%20ProjectInfo.FullName%20%7D%7D.Web/Pages/%7B%7B%20EntityInfo.NamespaceLastPart%20%7D%7D/%7B%7B%20EntityInfo.Name%20%7D%7D/Index.cshtml.sbntxt

Refer to: https://github.com/abpframework/abp/blob/dev/modules/identity/src/Volo.Abp.Identity.Web/Pages/Identity/Roles/Index.cshtml

1. Needs "PageLayout" generation: `Title`, `BreadCrumb` and `MenuItemName` configurations in PageLayout.Content.

1. "\<h2>\</h2>" should be "\<abp-card-title>\</abp-card-title>".

1. "size-md" should be `_6` and `_6`.

1. Creation button needs permission check. Relate to: https://github.com/EasyAbp/AbpHelper.CLI/issues/33 .

1. "\<abp-table>" may need `class="nowrap"`.

**Comment by gdlcf88** [comment]
```
@inject IPageLayout PageLayout
@{
    ViewBag.PageTitle = "Products";

    PageLayout.Content.Title = L["Product"].Value;
    PageLayout.Content.BreadCrumb.Add(L["Menu:ProductManagement"].Value);
    PageLayout.Content.MenuItemName = L["Menu:ProductManagement"].Value;
}
```

**Comment by wakuflair** [comment]
- [x] 1 
- [x] 2
- [x] 3
- [ ] 4 is related to #33, which will be resolved in [v0.3](https://github.com/EasyAbp/AbpHelper.CLI/milestone/2)
- [x] 5

Close for now.
